From: Mauro Carvalho Chehab <mchehab@redhat.com>
Subject: i7core_edac: Improve corrected_error_counts output for RDIMM
References: fate#311968
Git-commit: 3a3bb4a647db4cb2468641df8da2ee9491784a9a
Patch-mainline: v2.6.35-rc2


Signed-off-by: Thomas Renninger <trenn@suse.de>

Just cosmetics. instead of showing something like:

socket 0, channel 2dimm0: 1
dimm1: 0
dimm2: 0
socket 1, channel 2dimm0: 0
dimm1: 0
dimm2: 0

Show:

socket 0, channel 2 RDIMM0: 1 RDIMM1: 0 RDIMM2: 0
socket 0, channel 2 RDIMM0: 0 RDIMM1: 0 RDIMM2: 0

This is more synthetic and easier to parse.

Signed-off-by: Mauro Carvalho Chehab <mchehab@redhat.com>

---
 drivers/edac/i7core_edac.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: linux-2.6.32-SLE11-SP1/drivers/edac/i7core_edac.c
===================================================================
--- linux-2.6.32-SLE11-SP1.orig/drivers/edac/i7core_edac.c
+++ linux-2.6.32-SLE11-SP1/drivers/edac/i7core_edac.c
@@ -1071,9 +1071,9 @@ static ssize_t i7core_ce_regs_show(struc
 					pvt->udimm_ce_count[i][2]);
 		else
 			for (j = 0; j < NUM_CHANS; j++) {
-				count = sprintf(data, "socket %d, channel %d"
-						"dimm0: %lu\n"
-						"dimm1: %lu\ndimm2: %lu\n",
+				count = sprintf(data, "socket %d, channel %d "
+						"RDIMM0: %lu "
+						"RDIMM1: %lu RDIMM2: %lu\n",
 						i, j,
 						pvt->rdimm_ce_count[i][j][0],
 						pvt->rdimm_ce_count[i][j][1],
